### ISSUE TITLE
Bug Report: Insta-crashing on 64bit and variable crashing and asset failures on 32bit

### ISSUE THREAD

**Issue by CookidNA** [FunkinCrew/Funkin#561]
### [Newgrounds](https://www.newgrounds.com/portal/view/770371) or [Itch.io](https://ninja-muffin24.itch.io/funkin)? Specify below.
N/A
### Chrome (or a Chromium based browser, i.e. Edge, Opera, etc.), Firefox, or Safari? Specify blow.
N/A
### Windows (`x86`), Windows (`x86_64`), Linux, or macOS? Specify below.
Windows 10 Home 64bit
## What version of the game are you using? Look in the bottom left corner of the main menu.
0.2.7.1
## Have you identified any steps to reproduce the bug? If so, please describe them below. Use images if possible.
Seen below.
## Please describe your issue. Provide extensive detail and images if possible.

Upon opening the game in rar file, extracted, on seperate harddrive D or F, new download, deleting appdata cache, allowing through firewall, with administrator, or compatibility mode (any combination of these), the following happens:

On 64bit version: Nearly instant crash. Freezes at the start, and then 2 of the haxeflixel wings appear and crashes instantly after. I have nothing else to say, this is all it does.

On 32bit version: I'll start with the weeks. Usually, after playing 1 song the next song will crash the game, and assets wont load and be black squares, but there are some week exceptions, so they need to be taken on a 1 by 1 basis:
Tutorial: Loads fine, all assets intact and goes back to menu on finish. Cannot load another song afterwards.
Week 1: Loads a bit slower, all assets intact. Loads song 2 but crashes afterwards, no 3rd.
Week 2: Can't even load the first song. Goes to black screen, nothing happens, crash. 
Week 3: Speakers asset does not load. (Image: https://prnt.sc/10hnkuc) Can't tell if girlfriend asset is loaded, because the speaker asset is so big that the black square covers the entire area. Plays first song but crashes on entering 2nd.
Week 4: Somehow, miraculously, entirely intact. All assets good, all songs loaded, goes to menu. Cannot re-enter any week, including itself though. After each song there's also no black transition, just freezes and goes to next song.
Week 5: Same as week 2.
Week 6: Just like week 4, except it doesn't make it to the menu. Crashes after song 3.

Freeplay:
All songs would not play a second song after finishing.
-Bopeebo: Fine, same as week.
-Fresh & Dadbattle: Loads without girlfriend/speaker asset. https://prnt.sc/10ho984
-Spookeez: will load* but with a cost at most assets. I'll let the picture explain for me: https://prnt.sc/10hnlvz
-South: Crashes just like opening week 2 does.
-Pico: Fine, still missing girlfriend/speaker asset though.
-Philly & Blammed: Loads, but also missing boyfriend asset in addition to girlfriend/speaker asset. https://prnt.sc/10ho26c
-Week 4 songs will rarely** not load the mom asset. https://prnt.sc/10hnu17 But otherwise they load and play fine.
-Cocoa and Eggnog: Crashes.
-Winter horrorland: Will load ONLY if you scroll up and get to the bottom of the list and then go to it. Going down loads too many song oggs and it won't load then.
-Senpai, Roses & Thorns: The savior of the weeks! These in freeplay function absolutely perfectly. You can retry after death, you can leave/beat the song and play it again or play another pixel song, as much as you want. Ofcourse you can't go to a normal art style song afterwards though.

Dying won't crash conditionally***, but retrying will crash.

*I've found that it sometimes doesn't load though, seems that if you don't wait for the song list to finish its position or load too many songs in the freeplay list it won't load and will crash.
**I cannot tell but it seems this asset fail only happened on the first attempt to load a week 4 song.  I cannot reproduce it.
***Some songs will allow you to die with absolutely no freezing, but some songs will crash on death. Heres the list for freeplay:
Bopeebo, fresh, dadbattle: Fine, can die
spookeez: Fine, can die.
pico, philly, blammed: Fine, can die (but philly and blammed missing the boyfriend asset means their death screen is totally black)
satin, high, milf: Can't. Freezes for a while then crashes
winter horrorland: Can die, but takes a second
Week 6 songs: as stated above, can die and CAN RETRY.

So that's everything I tested. My perspective on this is that there is obvious loading errors (atleast on my end) that cause the game to crash. Mostly the game crashes after 1 song, assets are sometimes black, the freeplay list lags between each new song you hover over in the list, etc etc.
I don't know what on my end would cause this to happen, but some info is that my windows is quite corrupted. It never interferes with games, only problem i've gotten is sticky notes bugs the hell out and I can't view my browser passwords because the pin authenticator crashes, but nothing relating to games or file saving.

Just for good measure, I'll include some of my pc specs here but I feel they are unnecessary. I downloaded this on my sister's 2012 laptop and it ran just fine besides some fps lag.

CPU: A8-5600K APU
8GB RAM
Sapphire Radeon HD 7770 (https://www.gpuzoo.com/GPU-Sapphire/Radeon_HD_7770_1GB_GDDR5.html)

**Comment by lolBoi9696** [comment]
Why does this happen to people it doesn't happen to me try reinstalling Ur os

**Comment by CookidNA** [comment]
Issue was somehow fixed.
Miraculously, by another mod. I did no other changes, restarts, etc. I had just tried the game before this, but after running the Saturday Night SiIva (siivagunner rips) mod, everything got fixed. No more crashes on main game, or on vswhitty.

I have no idea how to attempt to guess what happened or how the mod magically fixed everything, but my guess is faulty gamesave data, IF there is any that isn't inside appdata. Because I deleted the folder in appdata many times yielding no result.

So, I have really nothing to add. I guess if this is an issue ever brought up again, recommend them to run silva mod? Maybe study what code is in the silva mod? No clue, but my issues are gone now. Closed.
